pkg/kube/snapshot/snapshot_alpha.go

```diff
@@ -278,6 +282,14 @@ func (sna *SnapshotAlpha) CreateFromSource(ctx context.Context, source *Source,
 	return sna.WaitOnReadyToUse(ctx, snapshotName, namespace)
 }
 
+func getAnnotation(annotations map[string]interface{}, key string) map[string]string {
+	if val, ok := annotations[key]; ok {
+		return val.(map[string]string)
+	}
+
+	return nil
+}
```

Comment:
This is confusing for me, are annotations `,` separated? Because we are getting annotation value for either 
```
SnapshotContentAnnotation         = "snapshotcontent"
	SnapshotAnnotation                = "snapshot"
```
and then just returning the value? 🤔

Reply:
Since the annotation will be in below format:
```
testAnnotation := map[string]interface{}{
		SnapshotAnnotation : map[string]string{
			"testannotation" : "value",
			"testannotation1": "value1",
		},
		SnapshotContentAnnotation : map[string]string{
			"testannotation2" : "value2",
			"testannotation3": "value3",
		},
	}
```
`getAnnotation` is function to get interface value on the basis of key `SnapshotAnnotation` or `SnapshotContentAnnotation`.